dataconfig/parsers.py

```diff
@@ -330,6 +367,25 @@ def _spec_to_type(
             )
         )
     )  # chain dict.items() and create namespace
+    '''
```

Comment:
Don't comment out old code like this.  That's why we have version control.  Just remove it.